tools/roadmap/validate_dag.py

```
@@ -0,0 +1,119 @@
+def gh_json(args: Sequence[str]) -> dict:
+    out = run(["gh", *args])
+    return json.loads(out)
+
+def main():
+    if not MMD.exists():
+        print("validate_dag: missing docs/ROADMAP-DAG.mmd", file=sys.stderr)
+        return 1
+    text = MMD.read_text(encoding='utf-8')
+    nodes, edges_hard, _ = parse_mmd(text)
+    # Map node id -> issue number
+    node_to_issue = {n: num for (n,num) in nodes}
```

Ruff (0.14.3)

Unnecessary dict comprehension (rewrite using `dict()`)

Rewrite using `dict()`

(C416)